src/widget/weffectselector.cpp

```diff
@@ -57,15 +57,13 @@ void WEffectSelector::populate() {
                                                        width() - 2);
         addItem(elidedDisplayName, QVariant(pManifest->id()));
 
-        // NOTE(Be): Using \n instead of : as the separator does not work in
-        // QComboBox item tooltips.
-        // TODO(Be): Check if this is also the case with Qt5.
-        //: %1 = effect name; %2 = effect description
-        QString description = tr("%1: %2").arg(pManifest->name(),
-                                               pManifest->description());
+        QString name = tr("%1").arg(pManifest->name());
+        QString description = tr("%1").arg(pManifest->description());
         // The <span/> is a hack to get Qt to treat the string as rich text so
         // it automatically wraps long lines.
-        setItemData(i, QVariant("<span/>" + description), Qt::ToolTipRole);
+        // <b> makes the effect name bold.
+        setItemData(i, QVariant("<span/><b>" + name + "</b><br/>"
+                + description), Qt::ToolTipRole);
```

Comment:
If we start with `<b>` I don't think we need the span anymore. Also, shouldn't we use QStringLiterals here?